js/main.js

```diff
@@ -0,0 +1,127 @@
+$(document).ready(function () {
+
+    $("#loginForm").submit(function (event) {
+
+        var email = $("#email").val();
+        var password = $("#pwd").val();
+
+        $.ajax({
+            type: "POST",
+            data: {
+                action: 'authenticateUser',
+                table: 'test_table1',
+                email: email,
+                password: password
+            },
+            url: "ajax_php/actions.php",
+            beforeSend: function () {
+                console.log('sending request ...');
+            },
+            dataType: "html",
+            async: true,
+            success: function (data) {
+
+                var result = JSON.parse(data);
+                const token = result['token'];
+
+                if (result['error']) {
+
+                    trackLogin(false);
+
+                    $('#email').val('');
+                    $('#pwd').val('');
+                    $('#error').text(result['error']['message']);
+
+                } else {
+
+                    trackLogin(true);
+
+                    $("#loginForm").hide();
+                    $("#results").removeClass("hidden");
+
+                    $.ajax({
+                        type: "GET",
+                        data: {
+                            action: 'retrieveForms',
+                            token: token
+                        },
+                        url: "ajax_php/actions.php",
+                        beforeSend: function () {
+                            console.log('sending request ...');
+                        },
+                        dataType: "html",
+                        async: true,
+                        success: function (data) {
+
+                            var result = JSON.parse(data);
+
+                            $.each(result['data'], function (i, v) {
```

Comment:
preferabil este mereu sa folosesti un for in loc de $.each wrapper. codul scris asa este foarte neoptimizat.